.github/workflows/release_feature_freeze_ceremony.yaml

```diff
@@ -0,0 +1,90 @@
+name: Release - Feature Freeze Ceremony
+on:
+  workflow_dispatch:
+
+# The workflow needs the permission to push branches
+permissions:
+  contents: write
+  pull-requests: write
```

Comment:
if you do have a permissions block you need to include all permissions, so might be better to leave out